### PR TITLE
Make noarch

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This PR makes this feedstock noarch. From what I understand, the reason this was not yet the case is that the functionality is not available on Windows. Using the __unix virtual package that same restriction is implemented.

Ref: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html

### CODE REVIEW

File: recipe/meta.yaml
Comment (mrakitin) [inline recipe/meta.yaml:25]: I am not familiar with the `__unix` package. What does it do and why do we need it here?

**Review by mrakitin — COMMENTED** [review]
Thanks for your contribution, @BastianZim! I have a single question to resolve before it can get merged (see below).

File: recipe/meta.yaml
Comment (BastianZim) [reply recipe/meta.yaml:25]: It restricts the package to run only on Unix machines. The way I understand upstream, this package only works there so that makes sure that it cannot be installed on windows machines. 
When installing the package, the conda/mamba solver will inform the user that it is not available for this platform.

Comment (mrakitin) [reply recipe/meta.yaml:25]: Thanks for the explanation, @BastianZim. I can't find any help related to that at https://conda-forge.org/docs/search.html?q=__unix. Must be a hidden feature 🙂.

Maybe the @conda-forge/core folks could point us to the relevant docs regarding `__unix`? I wonder if there are other types of `__<...>` things that could be useful for other feedstocks.

Comment (BastianZim) [reply recipe/meta.yaml:25]: It's in the docs I linked above :) 

https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html

Comment (mrakitin) [reply recipe/meta.yaml:25]: My bad 🤦. Sorry for the noise. Looks good to me!

**Review by mrakitin — APPROVED** [review]

**Comment by mrakitin** [comment]
@BastianZim, want to become a co-maintainer? 😉

File: recipe/meta.yaml
Comment (BastianZim) [reply recipe/meta.yaml:25]: No worries. Can you also merge the PR please?

**Comment by BastianZim** [comment]
Sure, I can help out. 😄

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by mrakitin** [comment]
Thanks a lot! I am not sure how to exclude the CircleCI builds (they don't seem to be supported), but it's not a blocker.

**Comment by BastianZim** [comment]
You can ignore the CircleCI, that's a leftover from before. At some point a bot should come by and clean that up.

**Comment by mrakitin** [comment]
Welcome to the team, @BastianZim! Thanks again for your help.

**Pull request merged**
